Calendar/CalLogHelpers/TimelineFunctions.ps1

```diff
@@ -413,6 +422,13 @@ function TimelineRow {
                         [array] $Output = "Transport $($CalLog.TriggerAction)d a Meeting Cancellation based on changes by [$($CalLog.SenderSMTPAddress)]."
                     }
                 }
+                ResourceBookingAssistant {
+                    if ($callog.TriggerAction -eq "MoveToDeletedItems") {
```

Comment:
Triggering a spellcheck issue in build pipeline

Reply:
Thanks.